core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -451,6 +529,45 @@ private ExprValue exprDate(ExprValue exprValue) {
     }
   }
 
+  /**
+   * DateTime implementation for ExprValue.
+   *
+   * @param dateTime ExprValue of String type.
+   * @return ExprValue of date type.
+   */
```

Comment:
Please, describe both params in javadoc

Reply:
Second parameter added to java doc. Java doc also added to exprDateTimeNoTimezone function.